template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/impala/ImpalaVolumeConfigProvider.java

```diff
@@ -38,20 +40,51 @@ public List<ApiClusterTemplateConfig> getRoleConfigs(String roleType, HostgroupV
                         .mapToInt(volume -> volume.getVolumeSize())
                         .min()
                         .orElse(0);
+                Integer volumeCount = 0;
+                Integer temporaryStorageVolumeCount = 0;
+                Integer temporaryStorageVolumeSize = 0;
```

Comment:
Can be primitive `int`s

Reply:
Yes, we can use primitive int. I will change it for both of them.